stdlib/2and3/plistlib.pyi

```diff
@@ -14,24 +14,16 @@ _D = TypeVar("_D", bound=mm)
 _Path = Union[str, Text]
 
 if sys.version_info >= (3, 9):
-    @overload
-    def load(fp: IO[bytes], *, fmt: Optional[PlistFormat] = ...) -> DictT[str, Any]: ...
-    @overload
-    def load(fp: IO[bytes], *, fmt: Optional[PlistFormat] = ..., dict_type: Type[_D]) -> _D: ...
-    @overload
-    def loads(value: bytes, *, fmt: Optional[PlistFormat] = ...) -> DictT[str, Any]: ...
-    @overload
-    def loads(value: bytes, *, fmt: Optional[PlistFormat] = ..., dict_type: Type[_D]) -> _D: ...
+    def load(fp: IO[bytes], *, fmt: Optional[PlistFormat] = ..., dict_type: Type[_D] = ...) -> Any: ...
```

Comment:
Sorry, one more nit: we no longer need the typevar, so we can just inline the type directly (and delete mm / _D above)
```suggestion
    def load(fp: IO[bytes], *, fmt: Optional[PlistFormat] = ..., dict_type: Type[MutableMapping[str, Any]] = ...) -> Any: ...
```